innersource-guidance-group.md

```diff
@@ -0,0 +1,43 @@
+## Title
+InnerSource Guidance Group
+
+## Patlet
+A company may require a mechanism to set broad governance and behavioral guidelines as part of the rollout and sustaining needs of an InnerSource initiative.
```

Comment:
Patlet should summarize the solution (in addition to the problem).

Reply:
Changed the pattern like this (a bit more liberally):

> A highly divergent set of development standards in different teams can slow down collaboration betweens these teams. A InnerSource Guidance Group that establishes broad governance and behavioral guidelines can help to reduce these frictions.